doc/content/manuals/user/index.md

```diff
@@ -1,4 +1,126 @@
+
+#### Prescribed Acceleration
+
+The preset acceleration boundary condition can be used to apply an acceleration time history to a boundary. The preset acceleration boundary condition integrates the given acceleration time history to get the displacement using Newmark-beta method. This displacement is then applied as a kinematic displacement boundary condition. Syntax is the same as prescribing a displacement boundary condition but with type = PresetAcceleration and the function describing time vs acceleration data instead of time vs displacement.
+
+### Periodic Boundary Conditions
+
+Periodic boundary conditions are used to constrain the nodes to move together in the specified directions. The following command is an example applied on the element presented in \ref{fig:single_element} and should be embedded into the BCs block segment as:
+
+!listing test/tests/materials/I_soil/HYS_darendeli.i start=Periodic end=top_x
+
+### Initial Stresses
+
+Initial stress command is available to apply the prescribed initial stresses to the finite element domain. If the user applies initial stress state that is equal to that of gravity loadings, upon transient analysis, no vibration occurs and the domain reaches equilibrium with zero strains. This eliminates the necessity of conducting separate analysis for free-field problems and allows the user to start directly with a dynamic analysis such as base shaking etc. The following command can be used within the material definition to activate the initial stresses in the material block:
+
+
+      initial_stress = '-4.204286 0 0  0 -4.204286 0  0 0 -9.810'
+
+The nine numerical values fill the stress tensor in the following order - xx, xy, xz, yz, yy, yz, zx, zy, and zz. The above example command activates -4.2025 units of stress along both xx and yy direction, and -9.81 units of stress along zz direction.  These stresses are the gravity stresses at the mid-depth of the element in \ref{fig:single_element} with density = 2. Therefore, the element will immediately start with gravity stress conditions without any elastic deformations or stress fluctuations.
+
+### Materials
+#### Linear elastic soil
+A linear elastic soil can be defined in MASTODON by defining a elasticity tensor using ComputeIsotropicElasticityTensor, stress calculator using ComputeLinearElasticStress, and a small strain calculator using ComputeSmallStrain. Density can be defined using the GenericConstantMaterial. An example of these input blocks is shown below:
+
+!listing test/tests/dirackernels/seismic_source/one_seismic_source.i start=Materials end=Executioner
+
+#### I-soil material model
+The nonlinear soil material model usage is demonstrated in the [Example 1] (/examples/index.md)
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```

Comment:
@aeslaughter, Please remove the above line from the user manual.

Reply:
Dang, thanks for catching this. My search failed me.